app/[locale]/institute/sections/library/library-committee/page.tsx

```diff
@@ -0,0 +1,73 @@
+import { Suspense } from 'react';
+
+import Heading from '~/components/heading';
+import Loading from '~/components/loading';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '~/components/ui';
+import { getTranslations } from '~/i18n/translations';
+import { db } from '~/server/db';
+
+export default async function libraryCommittee({
+  params: { locale },
+}: {
+  params: { locale: string };
+}) {
+  const libraryCommitteeData = await db.query.libraryCommittee.findMany({
+    with: {
+      facultyInfo: {
+        columns: {
+          id: true,
+          designation: true,
+        },
+        with: {
+          person: {
+            columns: {
+              name: true,
+            },
+          },
+        },
+      },
+    },
+  });
+  const text = (await getTranslations(locale)).Section.Library.libraryCommittee;
+
+  return (
+    <section className="container">
+      <Heading
+        glyphDirection="dual"
+        heading="h2"
+        text={text.libraryCommitteeTitle}
+        href="#library-committee"
+        id="library-committee"
+      />
+      <Suspense fallback={<Loading />}>
+        <Table>
+          <TableHeader>
+            <TableRow>
+              <TableHead>{text.srNo}</TableHead>
+              <TableHead>{text.name}</TableHead>
+              <TableHead>{text.generalDesignation}</TableHead>
+              <TableHead>{text.libraryCommitteeDesignation}</TableHead>
+            </TableRow>
+          </TableHeader>
+          <TableBody>
+            {libraryCommitteeData.map((entry, idx) => (
```

Comment:
```suggestion
            {libraryCommitteeData.map((entry, index) => (
```
As a general rule of thumb, avoid abbreviations in variable names.